lib/sam/common/pmc.c

```diff
@@ -44,20 +44,29 @@ void pmc_plla_config(uint8_t mul, uint8_t div)
 
 void pmc_peripheral_clock_enable(uint8_t pid)
 {
+#if defined(SAM3N) || defined(SAM3U)
+	//SAM3N and SAM3U only have one Peripheral Clock Enable Register
+	PMC_PCER = 1 << pid;
+#else
```

Comment:
the same register has a different name on SAM3N and SAM3U and the register for the enable bits 32 to 63 isn't there. checking for the presence PMC_PCER2 should also do the trick